### PR TITLE
Config variable handling update

### LINKED ISSUE #301: Update chart secret resource to use stringData
### Problem

Currently the secret resource utilizes `data`.
This results in multiple extra base64 conversions.

### Solution

Use `stringData` instead of `data` to avoid all the extra base64 conversions.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by rustyShacklefurd**
**Description**:

**Related issue(s)**:

Fixes #301 

**Notes for reviewer**:
Partial `helm template` render of the chart against the helm-chart/environments/minikube.yaml values
```
---
# Source: hedera-json-rpc-relay/templates/secret.yaml
apiVersion: v1
kind: Secret
metadata:
  labels:
    helm.sh/chart: hedera-json-rpc-relay-0.6.0-SNAPSHOT
    app.kubernetes.io/name: hedera-json-rpc-relay
    app.kubernetes.io/instance: test
    app.kubernetes.io/version: "0.6.0-SNAPSHOT"
    app.kubernetes.io/managed-by: Helm
  name: test-hedera-json-rpc-relay
type: Opaque
stringData:
  OPERATOR_ID_MAIN:
  OPERATOR_KEY_MAIN:
  OPERATOR_ID_ETH_SENDRAWTRANSACTION: ""
  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: ""
---
# Source: hedera-json-rpc-relay/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: test-hedera-json-rpc-relay
  labels:
    app:  hedera-json-rpc-relay

    helm.sh/chart: hedera-json-rpc-relay-0.6.0-SNAPSHOT
    app.kubernetes.io/name: hedera-json-rpc-relay
    app.kubernetes.io/instance: test
    app.kubernetes.io/version: "0.6.0-SNAPSHOT"
    app.kubernetes.io/managed-by: Helm
data:
  HEDERA_NETWORK: {"192.168.1.10:50211":"0.0.3","192.168.1.11:50211":"0.0.4"}
  CHAIN_ID:
  MIRROR_NODE_URL:
  LOCAL_NODE:
  SERVER_PORT:
  CHAIN_ID:
```

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: helm-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+data:
+{{- if .Values.config.local.HEDERA_NETWORK }}
+  HEDERA_NETWORK: {{ .Values.config.local.HEDERA_NETWORK | toJson }}
+{{- else if .Values.config.hosted.HEDERA_NETWORK }}
+  HEDERA_NETWORK: {{ .Values.config.hosted.HEDERA_NETWORK }}
+{{- else }}
+  HEDERA_NETWORK: {{ required "A valid HEDERA_NETWORK must be present in either .Values.config.local.HEDERA_NETWORK or .Values.config.hosted.HEDERA_NETWORK " (or .Values.config.local.HEDERA_NETWORK .Values.config.hosted.HEDERA_NETWORK) }}
+{{- end }}  
+  OPERATOR_ID_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_ID_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
+  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_KEY_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
```
Comment (Nana-EC) [inline helm-chart/templates/configmap.yaml:17]: Given the nature of the details shouldn't this key also be a secret?

Comment (rustyShacklefurd) [reply helm-chart/templates/configmap.yaml]: Moved these two items to secrets and out of configmaps

File: helm-chart/environments/minikube.yaml [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
-  
\ No newline at end of file
+
+config:
+  # This value should be in decimal, not hexadecimal format.
+  CHAIN_ID: ""
+  MIRROR_NODE_URL: ""
+  LOG_LEVEL: ""
+  local:
+    HEDERA_NETWORK: {"192.168.1.10:50211":"0.0.3","192.168.1.11:50211":"0.0.4"}
```
Comment (Nana-EC) [inline helm-chart/environments/minikube.yaml:10]: Should be local node version 
```suggestion
    HEDERA_NETWORK: {"127.0.0.1:50211":"0.0.3"}
```

File: helm-chart/values.yaml [hunk 3]
```diff
@@ -84,13 +84,20 @@ ports:
...
-  OPERATOR_ID_ETH_SENDRAWTRANSACTION: ""
-  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: ""
   # This value should be in decimal, not hexadecimal format.
   CHAIN_ID: ""
   MIRROR_NODE_URL: ""
   LOG_LEVEL: ""
+  # Use config.local.HEDERA_NETWORK when running against a locally hosted hedera network
+  # local:
+    # config.local.HEDERA_NETWORK should configured as a json set - {"$IPv4_ADDR_1:$PORT_1":"$ACCOUNT_ID_1","$IPv4_ADDR_2:$PORT_2":"$ACCOUNT_ID_2"}
+    # HEDERA_NETWORK: {"192.168.1.10:50211":"0.0.3","192.168.1.11:50211":"0.0.4"}
```
Comment (Nana-EC) [inline helm-chart/values.yaml:94]:
```suggestion
    # HEDERA_NETWORK: {"192.168.1.10:50211":"0.0.3","192.168.1.11:50211":"0.0.4"}
```
```suggestion
    # HEDERA_NETWORK: {"127.0.0.1:50211":"0.0.3"}
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG.
Should just use `127.0.0.1` for local examples

File: helm-chart/environments/minikube.yaml (hunk 2 above)
Comment (rustyShacklefurd) [reply helm-chart/environments/minikube.yaml]: updated

File: helm-chart/values.yaml (hunk 3 above)
Comment (rustyShacklefurd) [reply helm-chart/values.yaml]: updated

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**